src/eolab/rastertools/processing/sliding.py

```diff
@@ -66,7 +66,7 @@ def compute_sliding(input_image: str, output_image: str, rasterprocessing: Raste
                 raise ValueError(f"Invalid bands, all values are not in range [1, {src.count}]")
 
             # setup profile for output image
-            profile.update(driver='GTiff',
+            profile.update(driver='GTiff', compress='lzw',
```

Comment:
While this change is minimal and it's really a good practice to always use compression, maybe it should somehow be configurable.

@queyruto what do you think? Function kwarg? Or using a rasterprocessing attribute if it exists like for dtype?

Reply:
Yes, it is a good idea to add the compression as a rasterprocessing's attribute like dtype.

Reply:
So @erblanm add the compression as rasterprocessing attribute, like dtype.

Reply:
Ok I can do that